test/test_wal_file_deleter.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) 2021 Aiven, Helsinki, Finland. https://aiven.io/
+
+import time
+from queue import Queue
+
+import mock
+import pytest
+
+from pghoard import metrics
+from pghoard.compressor import WALFileDeleterThread
```

Comment:
I don't think we have a linter here but the import order seem a bit chaotic.

Reply:
there is a linter but no isort. This order is from pycharm -> code -> optimize imports